PyInstaller/building/makespec.py

```diff
@@ -53,6 +55,15 @@ def make_path_spec_relative(filename, spec_dir):
     )
 
 
+def add_data_or_binary(string):
+    # Or should we use a regex here?
+    if string.count(add_command_sep) != 1:
```

Comment:
`add_command_sep in string`
You are not interested in the number of occurrences :-)